src/components/Layout.tsx

```diff
@@ -41,6 +42,7 @@ const Layout: React.FC<LayoutProps> = ({ title, children }) => {
         <script
           type="text/javascript"
           src={withAssetPrefix('/js/docsearch.min.js')}
```

Comment:
it looks like this file was removed in this pr?

Reply:
Good eye ! It was just minified not deleted though

Reply:
ahhh i see!